dropwizard-migrations/src/test/java/io/dropwizard/migrations/DbMigrateCustomSchemaTest.java

```diff
@@ -0,0 +1,46 @@
+package io.dropwizard.migrations;
+
+import com.google.common.collect.ImmutableMap;
+import net.jcip.annotations.NotThreadSafe;
+import net.sourceforge.argparse4j.inf.Namespace;
+import org.junit.Before;
+import org.junit.Test;
+import org.skife.jdbi.v2.DBI;
+import org.skife.jdbi.v2.Handle;
+
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@NotThreadSafe
+public class DbMigrateCustomSchemaTest extends AbstractMigrationTest {
+
+    private DbMigrateCommand<TestMigrationConfiguration> migrateCommand = new DbMigrateCommand<>(
+        TestMigrationConfiguration::getDataSource, TestMigrationConfiguration.class, "migrations-custom-schema.xml");
+    private TestMigrationConfiguration conf;
+    private String databaseUrl;
+
+    @Before
+    public void setUp() throws Exception {
+        databaseUrl = "jdbc:h2:" + createTempFile();
+        conf = createConfiguration(databaseUrl);
+    }
+
+    @Test
+    public void testRunMigrationWithCustomSchema() throws Exception {
+        String schemaName = "customschema";
+        try (Handle handle = new DBI(databaseUrl, "sa", "").open()) {
```

Comment:
I think we could extract `DBI` as a variable and then use  Java-8 friendly method `useHandle`:
```java
DBI dbi = new DBI(databaseUrl, "sa", "");
dbi.useHandle(h -> h.execute("create schema " + schemaName));
```